source/material_model/interface.cc

```diff
@@ -747,6 +747,46 @@ namespace aspect
       return vs;
     }
 
+    namespace
```

Comment:
Which material models will use this property? I am asking, because I would prefer this to be moved in some place other than the interface (it is only relevant for plastic material models). If there is only one (or several, but they are all derived from one base model), then please move all this code into the header and source file of that class (see e.g. the handling of `MeltOutputs` in melt.h and melt.cc).